src/commands/setup.rs

```diff
@@ -62,7 +63,7 @@ impl Setup {
         let mut response: HashMap<String, types::StatusBlock> = HashMap::new();
 
         // Perform per-network setup
-        for (net_name, network) in network_options.network_info {
+        for (net_name, network) in network_options.network_info.iter() {
```

Comment:
Is `iter` needed here since i cant the change in `network_info`

Reply:
yes, it is needed